### PR TITLE
Refresh icon cache

### LINKED ISSUE #37: Wrong icon after installation
screenshot from 2017-05-18 22 13 16

When explicitly searching for "Nimbus" is OK, though.

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
This should fix #37

### CODE REVIEW

File: meson/post_install.py [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+ICONS_DIR = path.join(PREFIX, 'share', 'icons', 'hicolor')
 
-if not os.environ.get('DESTDIR'):
-	print('Compiling gsettings schemas...')
-	subprocess.call(['glib-compile-schemas', schemadir])
+if not environ.get('DESTDIR'):
+    print('Compiling gsettings schemas...')
+    call(['glib-compile-schemas', SCHEMA_DIR])
+    print('Refreshing icon cache...')
+    call(['gtk-update-icon-cache', '-qtf', ICONS_DIR])
```
Comment (danrabbit) [inline meson/post_install.py]: These options seem potentially dangerous

Comment (bil-elmoussaoui) [inline meson/post_install.py]: You prefer without any option or just keeping the force?

Comment (danrabbit) [inline meson/post_install.py]: Probably quiet is the only safe one. Ignoring existing files and force overwriting sounds bad

**Comment by bil-elmoussaoui** [comment]
Any news on this?

**Comment by bil-elmoussaoui** [comment]
@danrabbit Is there anything that should I do before reviewing/merging this one?

**Comment by NathanBnm** [comment]
```python
#!/usr/bin/env python3
import os
import subprocess

schemadir = os.path.join(os.environ['MESON_INSTALL_PREFIX'], 'share', 'glib-2.0', 'schemas')
iconchachedir = os.path.join(os.environ['MESON_INSTALL_PREFIX'], 'share', 'icons', 'hicolor')

if not os.environ.get('DESTDIR'):
    print('Compiling gsettings schemas…')
    subprocess.call(['glib-compile-schemas', schemadir])

    print('Rebuilding desktop icon cache…')
    subprocess.call(['gtk-update-icon-cache', iconchachedir])
```

Isn't it more simple that way?

**Pull request closed without merging**